#! /bin/sh /usr/share/dpatch/dpatch-run
## 34_nebula_digitaltv_remote.dpatch by Mario Limonciello <superm1@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add support for Nebula Digitv remote (LP: #164867)

@DPATCH@
diff -urNad lirc~/doc/lirc.hwdb lirc/doc/lirc.hwdb
--- lirc~/doc/lirc.hwdb	2008-06-03 23:28:41.000000000 -0500
+++ lirc/doc/lirc.hwdb	2008-06-03 23:30:32.000000000 -0500
@@ -87,6 +87,7 @@
 KNC ONE TV Station (-/SE/PRO/RDS);knc_one;lirc_dev lirc_i2c;hw_default;knc_one/lircd.conf.knc_one;
 Lenco MXTV-9578 CP (card=50);pixelview_pak;devinput;hw_default;pixelview/lircd.conf.playtv_pro;
 Miro PCTV serial port receiver;pctv;none;hw_pinsys;pinnacle_systems/lircd.conf.pctv;
+Nebula Digitv;leadtek;devinput;hw_default;nebula/digitv.conf;
 Phoebe Tv Master + FM (card=22);cph06x;devinput;hw_default;cph03x/lircd.conf.cph03x;
 Pinnacle Systems PCTV Sat receiver;pctv;none;hw_pinsys;pinnacle_systems/lircd.conf.pctv;
 Pixelview PlayTV MPEG2;pixelview_pro;devinput;hw_default;pixelview/lircd.conf.playtv_pro;
diff -urNad lirc~/remotes/nebula/digitv.conf lirc/remotes/nebula/digitv.conf
--- lirc~/remotes/nebula/digitv.conf	1969-12-31 18:00:00.000000000 -0600
+++ lirc/remotes/nebula/digitv.conf	2007-11-24 09:30:38.000000000 -0600
@@ -0,0 +1,91 @@
+# Please make this file available to others
+# by sending it to <lirc@bartelmus.de>
+#
+# this config file was automatically generated
+# using lirc-0.7.2(any) on Sat May  6 12:48:36 2006
+#
+# contributed by J. Brusey
+#
+# brand:                       lirc-nebula.conf
+# model no. of remote control: Nebula Electronics PCI DigiTV Remote
+# devices being controlled by this remote: Nebula Electronics PCI DigiTV
+#
+
+begin remote
+
+  name nebula
+  bits           16
+  flags RC5|CONST_LENGTH
+  eps            30
+  aeps          100
+
+  one           917   851
+  zero          917   851
+  plead         930
+  pre_data_bits   16
+  pre_data       0x8001
+  gap          135605
+  toggle_bit      0
+
+
+      begin codes
+          TV                       0x0179
+          Aux                      0x0186
+          DVD/CD                   0x0185
+          Power                    0x0074
+          1                        0x0002
+          2                        0x0003
+          3                        0x0004
+          Picture                  0x016F
+          4                        0x0005
+          5                        0x0006
+          6                        0x0007
+          Audio                    0x0188
+          7                        0x0008
+          8                        0x0009
+          9                        0x000A
+          i                        0x0166
+          16:9                     0x00B7
+          0                        0x000B
+          14:9                     0x00B8
+          EPG                      0x016D
+          Exit                     0x00AE
+          Menu                     0x008B
+          Up                       0x0067
+          Left                     0x0069
+          OK                       0x001C
+          Right                    0x006A
+          Down                     0x006C
+          Ch+                      0x0192
+          +Vol                     0x0073
+          Ch-                      0x0193
+          -Vol                     0x0072
+          Red                      0x018E
+          Green                    0x018F
+          Yellow                   0x0190
+          Blue                     0x0191
+          SubT                     0x0172
+          AD                       0x00B9
+          Text                     0x0184
+          Mute                     0x0071
+          Rewind                   0x00A8
+          Stop                     0x0080
+          Play                     0x00CF
+          FastForward              0x00D0
+          Chapter                  0x00BA
+          Pause                    0x0077
+          Step                     0x00CF
+          Record                   0x00A7
+          PiP                      0x00BB
+          Zoom-                    0x004E
+          Zoom+                    0x004A
+          Capture                  0x00BC
+          Web                      0x00BD
+          e-Mail                   0x00D7
+          Phone                    0x00A9
+          PC                       0x0178
+      end codes
+
+end remote
+
+
